### ISSUE TITLE
Option to watch for added/removed torrents in directory

### ISSUE THREAD

**Issue by hongkongkiwi** [distribyted/distribyted#304]
Hi there,

I think it would be very practical to add/remove torrents without having to restart distribyted. I've setup a route with something like this:
```
routes:
  - name: blackhole
    torrent_folder: "/distribyted/torrents"
```

I'd like to have a `watch: true` option to watch for added/removed files and reload the torrents in the route appropriately for zero downtime updates.

**Comment by brilthor** [comment]
I agree this would be a very nice feature, it looks like there is something blocking it's implementation though according to: https://github.com/distribyted/distribyted/issues/5#issuecomment-1307076526
